src/validator.directive.js

```diff
@@ -225,6 +228,7 @@
             ctrl.$render();
             if (attrs.messageId) angular.element(document.querySelector('#' + attrs.messageId)).html('');
             else element.next().html('');
+            if ($validationProvider.validCallback) $validationProvider.validCallback(element);
```

Comment:
This fixes clean up of all bootstrap inputs when reset is called. Otherwise all elements are cleaned but they are marked red which makes no sense.